internal/lib/avi_api.go

```diff
@@ -94,8 +94,11 @@ func AviGetRaw(client *clients.AviClient, uri string, retryNum ...int) ([]byte,
 		utils.AviLog.Warnf("msg: Unable to fetch data from uri %s %v", uri, err)
 		checkForInvalidCredentials(uri, err)
 		apimodels.RestStatus.UpdateAviApiRestStatus("", err)
-		if aviError, ok := err.(session.AviError); ok && aviError.HttpStatusCode == 403 {
-			return nil, err
+		if aviError, ok := err.(session.AviError); ok {
+			if aviError.HttpStatusCode == 403 ||
+				strings.Contains(aviError.Error(), VSVIPNotFoundError) {
```

Comment:
The `vsvip not found` error was already present and it was not working as expected.

The caller was getting the error `msg: AviGetRaw retried 3 times, aborting` from the [L87](https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/pull/1033/files#diff-f64b581ab0c4e09ba7f29891ec8ce81926095cb5f13f8365a3e04a7d173eb9efR87) instead of the actual `vsvip not found`.